src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,140 @@
+- `description` - (Optional) Text that will be displayed above the form. Limited to 140 characters.
+
+## Example
+
+```javascript
+
+"elements": [
+    {
+        "type": "alert-rule-action",
+        "title": "Send Alerts to",
+        "settings": {
+        "type": "alert-rule-settings",
+        "uri": "/sentry/alert-rule",
+            "required_fields": [
+                {
+                    "type": "text",
+                    "label": "Channel",
+                    "name": "channel"
+                }
+            ]
+        }
+    }
+]
+```
+
+
+## Issue Alert Request Format
+
+When an issue alert fires, your service will need to read the settings from the alert payload. The `settings` are in `data.issue_alert.settings`. See the full [Issue Alert webhook documentation](/product/integrations/integration-platform/webhooks/#issue-alerts) for more information.
+
+### Payload
+
+```
+{
+    ...
+    "data": {
+        ...
+        "issue_alert": {
+            ...
+            "settings": [
+                {
+                    "name": "title",
+                    "value": "Ticket Title"
+                },
+                {
+                    "name": "description",
+                    "value": "Ticket Description"
+                }
+            ]
+        }
+    }
+}
+```
+
+## Metric Alert Request Format
+
+When a metric alert fires, your service will need to read the settings from the alert payload. The `settings` in this example are nested in `data.metric_alert.alert_rule.triggers[1].actions[0].settings`. There may be multiple `actions` and the index may not be 0 in your case. See the full [Metric Alert webhook documentation](/product/integrations/integration-platform/webhooks/#metric-alerts) for more information.
+
+### Payload
+
+
+```json
+{
+    “data”: {
+        ...
+        “metric_alert”: {
+            ...
+            “alert_rule”: {
+                ...
+                “triggers”: [
+                    ...
+                    {
+                        ...
+                        “actions”: [
+                            {
+                                ...
+                                “settings”: [
+                                    {
+                                        “name”: “title”,
+                                        “value”: “Critical Trigger”
+                                    },
+                                    {
+                                        “name”: “description”,
+                                        “value”: “Critical Description”
+                                    }
+                                ],
+
+                            }
+                        ]
+                    },
+                ],
+
+        },
+    },
+}
+```
+
+### Alert Rule Creation Error Handling
+
+When surfacing errors, we need to receive a JSON response with this shape:
+
+```javascript
+{message: string}
+```
+
+If data is not received as expected, nothing will be shown.
```

Comment:
This isn't true, if they return an error code (40x, 50x) We show a generic error message, you can probably just mention that instead.